MetaMask/metamask-extension
# Adding Permissions Tour & Empty State

## **Description**
This PR is for the conversion of the AllConnections component to the PermissionsPage component



## **Related issues**

Fixes:https://github.com/MetaMask/MetaMask-planning/issues/1989

## **Manual testing steps**

1. yarn && MULTICHAIN=1 yarn start
2. open fullscreen, add "#permissions" route to the end of the URL
3. disconnect all dapps and verify empty state after no dapps are connected.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
